### ISSUE TITLE
System.NullReferenceException

### ISSUE THREAD

**Issue by o2genum** [NeutroniumCore/Neutronium#60]
This happens on system shutdown, pretty rare. Relevantat least for 1.1.0.

```
System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.
   в Neutronium.WPF.HTMLApp.OnActivated(EventArgs e)
   в System.Windows.Application.WmActivateApp(Int32 wParam)
   в System.Windows.Application.AppFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   в MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   в MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   в System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   в System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   в System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   в MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
```

**Comment by David-Desmaisons** [comment]
THanks, I will investigate.

**Comment by David-Desmaisons** [comment]
Potential fix provided in version 1.3.0
